### PR TITLE
Refactored 

### PR DESCRIPTION
**Pull request by LopeKinz**
NEVER MERGE INTO MAIN BRANCH!
This Refactor was done by Sourcery AI.

Refactor was done to save Space, Resources and make the code more Readable.

STATS:
23 Files changed
139 additions and 170 deletions.

### CODE REVIEW

**Comment by LopeKinz** [comment]
AppVeyor Build falied due to older Python Version. This refactor is for python 3.8 or higher

File: examples/flask_app/src/backend/app.py [hunk 1]
```diff
@@ -9,6 +9,4 @@ def initialize():
 
 
 def do_stuff():
-    # do whatever you need to do
```
Comment (baseplate-admin) [inline examples/flask_app/src/backend/app.py:12]: let's keep this comment

File: examples/flask_app/src/backend/server.py [hunk 2]
```diff
@@ -53,13 +53,13 @@ def initialize():
...
+    response = (
+        {
             'status': 'ok',
         }
-    else:
-        response = {'status': 'error'}
-
+        if can_start
+        else {'status': 'error'}
+    )
```
Comment (baseplate-admin) [inline examples/flask_app/src/backend/server.py:62]: Old version looks more readable to me :)

**Review by baseplate-admin — CHANGES_REQUESTED** [review]
Hi, can we do these changes please ?

**Comment by LopeKinz** [comment]
Changes in progress

**Comment by LopeKinz** [comment]
changed!

**Comment by baseplate-admin** [comment]
Maybe we should remove python 3.7 support and move to python 3.8+

**Comment by r0x0r** [comment]
This refactoring is a mixed bag imo. Some parts are more readable, others not so much. I prefer simple code over compact one-liners.
@LopeKinz if you feel like making manual changes, I can go through this PR.

**Comment by baseplate-admin** [comment]
Note that this PR will also effectively require us to use python 3.8 as the minimum version

**Pull request closed without merging**